lab_1_classify_by_unigrams/main.py

```diff
@@ -34,6 +41,14 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if (not isinstance(tokens, list) or
+            not all(isinstance(stroka, str) for stroka in tokens) or not tokens):
+        return None
+    quantity = len(tokens)
```

Comment:
don't create a separate variable if you only use it once

Reply:
Fixed